helm_deploy/laa-estimate-eligibility/values/ccq-production.yaml

```diff
@@ -61,6 +61,7 @@ featureFlags:
   outgoingsFlow: ENABLED
   conditionalReveals: ENABLED
   earlyEligibility: ENABLED
+  legacyAssetReveals: NOT_ENABLED
```

Comment:
Is it `legacyAssetReveals` or `legacyAssetNoReveals` ??

Reply:
fixed